test/util/testutil.go

```diff
@@ -118,7 +119,7 @@ func RunTestDriver(t *testing.T, opts DriverOptions) (DriverOptions, csi.NodeCli
 		WriteKeypair:         opts.WriteKeypair,
 	})
 
-	d := driver.NewWithListener(lis, opts.Log, driver.Options{
+	d := driver.NewWithListener(lis, *opts.Log, driver.Options{
```

Comment:
Hm, previously it would be possible for a user to pass a nil pointer here. Now it isn't (and a user would need to create a Discard logger). Would it make sense to take a pointer here instead, and if it's nil, instantiate a discard logger automatically?

Reply:
That's already handled above, right? We create a TestLogger if `opts.Log` is nil.
I will leave it like this for now and you can improve it in a followup PR if necessary.